stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/OTPController.kt

```diff
@@ -29,7 +29,7 @@ class OTPController(val otpLength: Int = 6) : Controller {
      */
     fun onValueChanged(index: Int, text: String): Int {
         if (text == fieldValues[index].value) {
-            return 0
+            return 1
```

Comment:
returning 1 here would make sure that when the field has a char, inputting the same char would move focus to next field, otherwise it says at the same field